crates/zkvm/zkvm/src/proof.rs

```diff
@@ -95,6 +95,19 @@ impl ProofReceipt {
     pub fn public_values(&self) -> &PublicValues {
         &self.public_values
     }
+
+    /// Check if the proof receipt is empty
+    pub fn is_empty(&self) -> bool {
+        self.public_values.is_empty() && self.proof.is_empty()
+    }
```

Comment:
Would checking `self.proof.is_empty()` alone be sufficient?
Is it possible for public_values to be empty while proof remains non-empty?

Reply:
In the native mode, instead of setting `Proof` being equal to `PublicValues`, it made more sense to leave proof as empty and only have `PublicValues`. In that case we need to check both. 
`Proof` is empty in Native Mode. 
`PublicValues` is never empty, unless the receipt is empty.